public/locales/en/components.json

```
@@ -0,0 +1,135 @@
+{
+  "about": {
+    "modal-title": "About OpenSpace",
+    "img-alt-text": "OpenSpace logo",
+    "about-openspace-description": "OpenSpace is open-source interactive data visualization software designed to visualize the entire known universe and portray our ongoing efforts to investigate the cosmos.",
+    "fetching-version-number": "Fetching OpenSpace version...",
+    "custom-version-number": "Custom"
+  },
+  "collapsable-header": {
+    "chevron-aria-label": {
+      "open": "Open",
+      "close": "Close"
+    }
+  },
+  "color-edit": {
+    "error-invalid-color": "Invalid color"
+  },
+  "color-picker": {
+    "color-swatch-aria-label": "Open color edit"
+  },
+  "copy-to-clipboard-button": {
+    "tooltip": {
+      "copy": "Copy",
+      "copied": "Copied"
+    },
+    "label": "Copy"
```

This could potentially just be 
```suggestion
    "copy": "Copy",
    "copied": "Copied"
```
and use `"copy"` for the label as well

---

Alternatively, I think you should be able to reuse values from other keys, to avoid duplication

```suggestion
    "copy": "Copy",
    "copied": "Copied",
    "label": "$t(components.copy-to-clipboard-button.copy)"
```

Not sure what the preferred way to go is, though. Maybe it's easier to have it split up as you do now, even if it requires translating the same word multiple times

---

Yeah, I think it would be nice to have some kind of common keys if they are recurring. However, having this: `"label": "$t(components.copy-to-clipboard-button.copy)"` is going to be a lot of mental overhead to put on the users/community to understand. So then having to translate the same word twice is faster and easier

---

Definitely, especially since it's in the same place here

Something to think about in future places for common words though